app/controllers/components/course/courses_component.rb

```diff
@@ -35,7 +35,8 @@ def settings_sidebar_items
       settings_index_item,
       settings_components_item,
       settings_sidebar_item,
-      settings_notifications
+      settings_notifications,
+      settings_lesson_plan_item
```

Comment:
This belongs to the lesson plan component and should be placed as a settings_sidebar)item under `lesson_plan_component.rb` (similar to materials, videos etc).

Also, we generally have one settings tab per component. The 'types to hide/show' is one section of the settings. Whether lesson plan milestones should be collapsed by default might be another section. For this reason, use "Lesson Plan" instead of "Lesson Plan Items" as the tab title instead?